releng/platform-definitions/pom.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-   Copyright (c) 2018, 2022, Oracle and/or its affiliates. All rights reserved.
+   Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Did you mean to remove the 2018 from the copyright years here?

Reply:
Good catch.